daemon/inertiad/build/builder.go

```diff
@@ -317,15 +317,22 @@ func (b *Builder) herokuishBuild(d *Config, cli *docker.Client,
 	// Attach logs and report build progress until container exits
 	stop := make(chan struct{})
 	go containers.StreamContainerLogs(cli, resp.ID, out, stop)
-	status, err := cli.ContainerWait(ctx, resp.ID)
-	close(stop)
-	if err != nil {
-		return nil, err
+	statusCh, errCh := cli.ContainerWait(ctx, resp.ID, "")
```

Comment:
Seems like a bit of duplicate code is here. I think it would be nice to have a helper function instead.